### DISCUSSION TITLE
DIRECTION AXES

### DISCUSSION THREAD

**Discussion by Zuzumimi** [arkypita/LaserGRBL#1055] [Q&A]
bonjour, en cours d'utilisation de ma machine, les axes de direction ont changer et malgré les différentes solutions pour rétablir la bonne direction, je n'y arrive pas. dans configuration lorsque je change le 0 par 5 par exemple il ne se passe rien sur la ligne de commande. est-ce qu'il y a un moyen d’accéder à la configuration étendu afin de pouvoir y faire les corrections.
merci de votre prochains commentaires.
Michel

**Comment by StuartB4** [comment]
![llllllllllllllll](https://user-images.githubusercontent.com/16980026/85188886-eb934f00-b2a1-11ea-9b13-08f049049d29.jpg)
![mmmmmmmmmmmmmmmm](https://user-images.githubusercontent.com/16980026/85188908-15e50c80-b2a2-11ea-9967-6c4b3d541c3c.jpg)
